Resources/Prototypes/Entities/Clothing/Ears/headsets_prison.yml

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
These need to be in the DeltaV folders, rather than upstream folders.

Reply:
I will solve this the moment i remember how to find the place i need to put it

Reply:
this is fixed